lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -155,7 +155,7 @@ def remove_rule(self, rule_num: int) -> None:
     def find_rule(self, rule_num: int):
         return self.rules.get(rule_num)
 
-    def get_all_rules(self) -> List:
+    def get_all_rules(self) -> Dict:
```

Comment:
Do you know what kind of dict it is (e.g. `Dict[str, int]`)? In general it's good to be more specific with container types.

Reply:
Good point. It seems to me it's `Dict[int, List[Tuple[FlowMatch.Direction, str]]]`. Shall I add this or do you think it's too long?

Reply:
I think it's still good to have it even if it's not a very intuitive type... :)

Reply:
Done!